bin/ttmscalr

```diff
@@ -172,7 +172,7 @@ Main {
       scp               - Copy a remote file to local machine with SCP
       sftp              - Use SFTP to connect to a server
       ssh               - Use SSH to get a console on a server
-      terminate         - Terminate a server (a cure-all for sick dynos)
+      terminate         - Terminate a server, all servers in a role or a farm
```

Comment:
this makes it sound like you are terminating the servers in the farm, but that is different from terminating the farm itself.

"Terminate servers or a farm"

then for description it could be

"Terminate a specific server, all servers within a specific role, or an entire farm"